lib/engines/content_block_manager/app/workers/content_block_manager/delete_draft_content_blocks_worker.rb

```diff
@@ -0,0 +1,17 @@
+require "sidekiq/api"
+
+module ContentBlockManager
+  class DeleteDraftContentBlocksWorker < WorkerBase
+    sidekiq_options queue: :content_block_publishing
+
+    def perform
+      draft_content_editions = ContentBlockManager::ContentBlock::Edition.where(state: "draft").where("created_at < ?", 60.days.ago)
```

Comment:
wonder if there's a nicer way of writing this where clause?

Reply:
I think we can do:

```
ContentBlockManager::ContentBlock::Edition.draft.where("created_at < ?", 60.days.ago)
```